Casks/h/hyper-connect.rb

```diff
@@ -0,0 +1,35 @@
+cask "hyper-connect" do
+  version "1.0.3"
+  sha256 "c5ccba949f29f1fda54f9ebcf0f16a7c79e71a28732a50f4894406ef3a07727d"
+
+  url "https://cdn.cnbj1.fds.api.mi-img.com/mijia-ios-adhoc/hyperconnect/HyperConnect-#{version}.dmg",
+      verified: "mi-img.com/mijia-ios-adhoc/hyperconnect/"
+  name "Hyper Connect"
```

Comment:
```suggestion
  name "HyperConnect"
```

Based on the website, it appears that the official name is spelled without a space, i.e., 'HyperConnect'. We should rename the cask (and filename) to `hyperconnect` accordingly.